eng/pipelines/templates/steps/typespec-ci.yml

```diff
@@ -5,49 +5,6 @@ parameters:
   type: string
 
 steps:
-- script: |
-    RED='\033[0;31m'
-    exit_code=0
-    expected_npm_prefix=$(pwd)
-
-    # Log commands before running
-    set -x
-
-    pushd ${{parameters.Folder}} || exit_code=1
-
-    actual_npm_prefix=$(npm prefix)
-
-    if [ "$expected_npm_prefix" != "$actual_npm_prefix" ]; then
-      echo -e "\n${RED}ERROR: TypeSpec folders MUST NOT contain a package.json, and instead MUST rely on the package.json at repo root."
-      echo -e "${RED}Expected npm prefix: $expected_npm_prefix\n${RED}Actual npm prefix: $actual_npm_prefix\n"
-      exit_code=1
-    fi
-
-    # Pass "--no" to ensure npx does not install anything (TypeSpec compiler should already be installed)
-
-    if test -f main.tsp; then
-      npx --no tsp compile . --warn-as-error || exit_code=1
-    fi
-    if test -f client.tsp; then
-      npx --no tsp compile client.tsp --no-emit --warn-as-error || exit_code=1
-    fi
-
-    # Format parent folder to include shared files
-    npx tsp format ../**/*.tsp
-
-    popd
-
-    # If any files are added, changed, or deleted, print diff and return error
-    if [ -n "$(git status --porcelain)" ]; then
-      git status
-      git diff
-      exit_code=1
-    fi
-
-    # Revert any changes
-    git restore .
-    git clean -df
-
-    exit $exit_code
+- script: npx --no tsv ${{parameters.Folder}}
```

Comment:
Where do we expect this to be installed and built?

Reply:
the tsv tool would be built when we run npm ci at root